src/theory/logic_info.cpp

```diff
@@ -311,6 +311,11 @@ std::string LogicInfo::getLogicString() const {
         ss << "BV";
         ++seen;
       }
+      if (d_theories[THEORY_FF])
```

Comment:
I'd just remove this (this is the reason why the PR is failing CI).  For now, I would suggest enabling finite fields only when the logic is set to `ALL`. 

Logic strings are in need of refactoring (at the SMT-LIB standard level), so this code will likely be rewritten later.

Reply:
Hmm. So most of my CLI regression tests (yet un-merged) use the `QF_FF` logic string, so I left this code and just fixed the failing test.